aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,137 +1,11 @@
 ---
 remote_endpoint: ~
-name: "v1.10"
+name: "v1.10.1"
 proposals:
-  - name: step_1_increase_max_txn_gas
+  - name: step_1_upgrade_gas_schedule
     metadata:
-      title: "Increase max txn gas temporarily for framework upgrade"
-      description: "Increase max txn gas temporarily for framework upgrade"
+      title: "Gas Schedule Upgrade"
+      description: "Upgrade to version 15 to enable various gas-related changes (type creation gas, dependency gas & limits etc.)"
```

Comment:
where does the 15 being set?

Reply:
DefaultGas uses the `LATEST_GAS_FEATURE_VERSION` coded in Rust and therefore does not require an override of the feature version.

Reply:
Here's the generated proposal: https://gist.github.com/vgao1996/67aafe24d2926b0afb7816395327e9c7

Looks good to my naked eyes